pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go

```diff
@@ -547,6 +547,22 @@ func init() {
 			testName: "bz-kube-apiserver][invariant] alert/KubeAPIErrorBudgetBurn should not be at or above info",
 		}] = "history correction on kube update, expires 2022-01-24"
 	}
+
+	for _, jobName := range []string{
+		"periodic-ci-openshift-multiarch-master-nightly-4.10-upgrade-from-nightly-4.9-ocp-remote-libvirt-s390x",
+		"periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-aws-ovn-upgrade",
+		"periodic-ci-openshift-release-master-ci-4.10-e2e-aws-ovn-upgrade",
+		"periodic-ci-openshift-release-master-ci-4.10-e2e-azure-upgrade-single-node",
+		"periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-azure-ovn-upgrade",
+		"periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-azure-upgrade",
+		"periodic-ci-openshift-release-master-nightly-4.10-upgrade-from-stable-4.9-e2e-metal-ipi-upgrade",
+	} {
+		testsRequiringHistoryRewrite[testCoordinates{
+			jobName:       jobName,
+			testName:      "[sig-network] pods should successfully create sandboxes by other",
+			testSuiteName: "openshift-upgrade-test",
```

Comment:
this value needs to be set in all the skips above too, right?

Reply:
I wonder which value needs to be set though, openshift-tests or openshift-tests-upgrade. Possible some have moved.

Reply:
The "Test Comparison" page can tell you.

Reply:
It's snipping the end of the name, so I can't quite see it unless I'm missing something.

Reply:
Can use the export link in top right of the form to get a CSV that will show you the full suite name.

Reply:
Or just hover :)

Reply:
@DennisPeriquet so to be clear, check each of the tests above, look for them in the Test Comparison chart in data studio, find the ones not passing well (thus were added to ignore), verify their suite name, and add to these maps so each has a suite name.

Reply:
@deads2k when I put the testSuiteName in there, do I need to skip both the version that has the `"openshift-tests"`, `"openshift-tests-upgrade"` and `null` (I'm assuming `null` renders to `""`)?

Reply:
actually, the suite may also be something other than `openshift-tests` and `openshift-tests-upgrade`.

Reply:
i think you need to lookup which one is failing in the Test Comparison chart.

Reply:
yup, the "34.63% Working" one has Suite=openshift-tests-upgrade